superdesk/io/feed_parsers/nitf.py

```diff
@@ -207,28 +290,93 @@ def get_subjects(self, tree):
                 subjects.append({'name': subject_codes[qcode], 'qcode': qcode})
         return subjects
 
-    def get_keywords(self, docdata):
-        return [keyword.attrib['key'] for keyword in docdata.findall('key-list/keyword')]
+    def get_anpa_format(self, xml):
+        elem = xml.find("head/meta[@name='anpa-format']")
+        if elem is not None:
+            content = elem.get('content')
+            return content.lower() if content is not None else 'x'
 
-    def get_content(self, tree):
+    def parse_to_preformatted(self, element):
+        """
+        Extract the contnt of the element as a plain string with line enders
+        :param element:
+        :return:
+        """
         elements = []
-        for elem in tree.find('body/body.content'):
-            elements.append(etree.tostring(elem, encoding='unicode'))
+        soup = BeautifulSoup(element, 'html.parser')
+        for elem in soup.findAll(True):
+            elements.append(elem.get_text() + '\r\n')
         return ''.join(elements)
 
-    def get_norm_datetime(self, tree):
-        if tree is None:
-            return
+    def get_content(self, xml):
+        elements = []
+        for elem in xml.find('body/body.content'):
+            elements.append(etree.tostring(elem, encoding='unicode'))
+        content = ''.join(elements)
+        if self.get_anpa_format(xml) == 't':
+            if not content.startswith('<pre>'):
+                # convert content to text in a pre tag
+                content = '<pre>{}</pre>'.format(self.parse_to_preformatted(content))
+            else:
+                content = self.parse_to_preformatted(content)
+        return content
+
+    def get_format(self, xml):
```

Comment:
same thing, this part was not written by me (I have just moved it here to fix the merge conflict)